Lunch/TodaysLunchViewController.swift

```diff
@@ -7,20 +7,150 @@
 //
 
 import UIKit
+import Parse
+import MapKit
+import GoogleMaps
 
-class TodaysLunchViewController: UIViewController {
+class TodaysLunchViewController: UIViewController, MKMapViewDelegate {
+
+    class CustomPointAnnotation: MKPointAnnotation {
+        var imageName: String!
+    }
+
+    @IBOutlet weak var selectedPlaceLabel: UILabel!
+    @IBOutlet weak var selectedVotesLabel: UILabel!
+    @IBOutlet weak var mapView: MKMapView!
 
-    @IBOutlet weak var todaysLunchLabel: UILabel!
-    
     var textForLunchLabel = ""
-    
+    var textForVotesLabel = ""
+    var selectedIndex = -1
+    var placesClient: GMSPlacesClient?
+
+    let classNameKey = "Eateries"
+    let placeColumnKey = "place"
+    let voteColumnKey = "vote"
+    let image = UIImage(named: "PlaceIcon")
+
     override func viewDidLoad() {
         super.viewDidLoad()
-        todaysLunchLabel.text = textForLunchLabel
+        placesClient = GMSPlacesClient()
+
+        self.mapView.zoomEnabled = true
+        self.mapView.delegate = self
+
+        placesClient?.currentPlaceWithCallback({ (placeLikelihoodList: GMSPlaceLikelihoodList?, error: NSError?) -> Void in
+            if let error = error {
+                println("Pick Place error: \(error.localizedDescription)")
+                return
+            }
+
+            if let placeLikelihoodList = placeLikelihoodList {
+                let place = placeLikelihoodList.likelihoods.first?.place
+                if let place = place {
+                    var name = place.name
```

Comment:
This should be a `let`. This is true in a lot of other places and they become compiler warnings in Swift 2